pkg/sysinfo/sysinfo_linux.go

```diff
@@ -32,6 +33,11 @@ func New(quiet bool) *SysInfo {
 		sysInfo.AppArmor = true
 	}
 
+	// Check if Seccomp is supported.
+	if _, _, err := syscall.RawSyscall(syscall.SYS_PRCTL, syscall.PR_GET_SECCOMP, 0, 0); err != syscall.EINVAL {
```

Comment:
>              Since Linux 3.8, the Seccomp field of the /proc/[pid]/status
>              file provides a method of obtaining the same information,
>              without the risk that the process is killed; see proc(5).

Better safe than sorry? :innocent: